flavors/template-Exasol-all-python-3.12/flavor_base/language_deps/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM {{udfclient_deps}}
+
+RUN mkdir -p /build_info/packages/language_deps
+COPY language_deps/packages/apt_get_packages /build_info/packages/language_deps
+
+RUN /scripts/install_scripts/install_via_apt.pl --file /build_info/packages/language_deps/apt_get_packages --with-versions
+
+RUN /scripts/install_scripts/install_python3.12_pip.sh "pip == 25.2"
+
+COPY language_deps/packages/python3_pip_packages /build_info/packages/language_deps
+RUN /scripts/install_scripts/install_via_pip.pl --file /build_info/packages/language_deps/python3_pip_packages --python-binary python3.12 --with-versions --pip-needs-break-system-packages --ancestor-pip-package-root-path /build_info/packages
+
+ENV PYTHON3_PREFIX /usr
+ENV PYTHON3_VERSION python3.12
+
+RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.12 1 #Needed for Bazel build
```

Comment:
Need to install `python3` in the system.

Reply:
We potentially should also fix bazel here, that it uses the python from our environment variable. However, probably in a later PR.

Reply:
Created https://github.com/exasol/script-languages-release/issues/1199